src/fieldValidation.ts

```diff
@@ -24,6 +24,11 @@ const isString = ajv.compile({
 	type: 'string',
 });
 
+const isIdString = ajv.compile({
```

Comment:
I'm not going to change this in this PR, but I'm realizing that the `isFooString` vs `isFoo` is probably redundant.  In a future refactor we probably would just want to use `isId` from `types/Id.ts`

Reply:
I changed my mind...  Using `isId` here instead.